README.md

```diff
@@ -57,7 +57,7 @@ source  ~/.pyenv/versions/3.9.1/bin/virtualenvwrapper.sh
 
 `workon notifications-api`
 
-9. Install [Postgres.app](http://postgresapp.com/).
+9. Install [Postgres.app](http://postgresapp.com/) (version 12.6, not the latest).
```

Comment:
We run 11.9 in staging/production at the moment. May be good to update the version here while we're at it! https://github.com/cds-snc/notification-api/blob/0ede6a61b48289236d1873124965d2bc22a9b27b/docker-compose.yml#L5

Reply:
The Postgres app [currently allows](https://postgresapp.com/downloads.html) 13.2, 12.6, 11.11, 10.16 and older ones. Given that version 12.6 is the upper one which still works I would recommend it in the install process for the dev environment. We might suggest 11.11 to be closer to the staging/prod env.